Content.Shared/_RMC14/Xenonids/Construction/SharedXenoConstructionSystem.cs

```diff
@@ -199,19 +203,35 @@ private void OnXenoSecreteStructureAction(Entity<XenoConstructionComponent> xeno
         if (attempt.Cancelled)
             return;
 
+        var effectID = XenoStructuresAnimation + choice;
+        var coordinates = GetNetCoordinates(args.Target);
+        var entityCoords = GetCoordinates(coordinates);
+        EntityUid? effect = null;
+
+        if (_prototype.TryIndex(effectID, out var effectProto) && _net.IsServer)
+            effect = Spawn(effectID, entityCoords);
+
+        var ev = new XenoSecreteStructureDoAfterEvent(coordinates, choice, EntityManager.GetNetEntity(effect));
         args.Handled = true;
-        var ev = new XenoSecreteStructureDoAfterEvent(GetNetCoordinates(args.Target), choice);
         var doAfter = new DoAfterArgs(EntityManager, xeno, xeno.Comp.BuildDelay, ev, xeno)
         {
             BreakOnMove = true
         };
 
-        // TODO RMC14 building animation
-        _doAfter.TryStartDoAfter(doAfter);
+        _audio.PlayPredicted(xeno.Comp.BuildSound, entityCoords, xeno);
```

Comment:
In 13 this plays on structure building finish instead of start, it's probably better for us to do the same both in terms of when it sounds better to be played (gives feedback to the player that it was built), and also to stop sound spam from spam cancelling and constructing